yaml-tests/src/test/resources/initial-version/less-than-version-tests.yamsql

```diff
@@ -0,0 +1,58 @@
+#
+# less-than-version-tests.yamsql
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2021-2025 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This test should pass when run on version 3.0.18.0
+---
+schema_template:
+    create table t1(id bigint, col1 bigint, primary key(id))
+---
+test_block:
+  preset: single_repetition_ordered
+  tests:
+    -
+      - query: select * from t1;
+      - initialVersionLessThan: 3.0.19.0
+      - result: []
+      - initialVersionAtLeast: 3.0.19.0
+      - result: [{"foo"}]
+    -
+      - query: select * from t1;
+      - initialVersionLessThan: 3.0.19.0
+      - result: []
+      - initialVersionAtLeast: 3.0.19.0
+      - result: [{"foo"}]
+    -
+      - query: insert into t1(id, col1) values (1, 2), (2, 3);
+      - initialVersionLessThan: 3.0.19.0
+      - count: 2
+      - initialVersionAtLeast: 3.0.19.0
+      - count: 0
+    -
+      - query: select * from t1;
+      - initialVersionLessThan: 3.0.19.0
+      - unorderedResult: [{id: 1, col1: 2}, {id: 2, col1: 3}]
+      - initialVersionAtLeast: 3.0.19.0
+      - unorderedResult: []
+    -
+      - query: SHOULD ERROR;
+      - initialVersionAtLeast: 3.0.19.0
+      - unorderedResult: []
+      - initialVersionLessThan: 3.0.19.0
+      - error: '42601'
```

Comment:
I feel like this should be reversed to be consistent, although explicitly testing that other orderings work just as well seems good too, but in addition to the standard ordering.
```suggestion
      - initialVersionLessThan: 3.0.19.0
      - error: '42601'
      - initialVersionAtLeast: 3.0.19.0
      - unorderedResult: []
```